lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -36,6 +36,7 @@ function setAttributeMarker(metricName) {
       nodeLabel: getNodeLabel(elem), // eslint-disable-line no-undef
       // @ts-ignore - put into scope via stringification
       snippet: getOuterHTMLSnippet(elem), // eslint-disable-line no-undef
+      boundingRect: JSON.parse(JSON.stringify(elem.getBoundingClientRect())),
```

Comment:
Should we add a 'getBoundingRect' function to page-functions so that it's consistent with the other element data functions (getNodeLabel, getOuterHTMLSnippet, etc.)?

Reply:
good call, esp. is there needs to be a comment here.